force-app/main/service/ContactService.cls

```diff
@@ -176,6 +176,15 @@ public with sharing class ContactService {
         }
     }
 
+    /**
+     * @description Returns the Contact for the ContactId
+     * @param contactId Id the contact id
+     * @returns Contact
+     */
+    public Contact getContactForUser(Id contactId) {
```

Comment:
Again, name of the method is not descriptive of actual logic being implemented

Reply:
ok